frontend/src/components/modalContent/ViewIncidentModal.js

```diff
@@ -0,0 +1,288 @@
+    <div>
+      <h5>Incident Details</h5>
+      <div className="content--light-grey padding-small">
+        <div className="inline-flex padding-small">
+          <p className="field-title">Incident date</p>
+          <p>{formatDate(this.props.incident.incident_timestamp) || Strings.EMPTY_FIELD}</p>
+        </div>
+        <div className="inline-flex padding-small">
+          <p className="field-title">Incident time</p>
+          <p>{formatTime(this.props.incident.incident_timestamp) || Strings.EMPTY_FIELD}</p>
+        </div>
+        <div className="inline-flex padding-small">
+          <p className="field-title">Number of fatalities</p>
+          <p>{this.props.incident.number_of_fatalities}</p>
+        </div>
+        <div>
+          <div className="inline-flex padding-small">
+            <p className="field-title">Number of injuries</p>
+            <p>{this.props.incident.number_of_injuries}</p>
+          </div>
+          <div className="inline-flex padding-small">
+            <p className="field-title">Were emergency services called?</p>
+            <p>{this.props.incident.emergency_services_called ? "Yes" : "No"}</p>
+          </div>
+        </div>
+        <div className="inline-flex padding-small">
+          <p className="field-title">Brief description of incident</p>
+          <p>{this.props.incident.incident_description || Strings.EMPTY_FIELD}</p>
+        </div>
+
+        <div className="inline-flex padding-small">
+          <p className="field-title">Inspectors determination</p>
+          <p>
+            {this.props.incidentDeterminationHash[this.props.incident.determination_type_code] ||
+              Strings.EMPTY_FIELD}
+          </p>
+        </div>
+        {this.props.incident.determination_type_code === Strings.DANGEROUS_OCCURRENCE && (
+          <div className="padding-small">
+            <p className="field-title">
+              Which section(s) of the code applies to this dangerous occurrence?
+            </p>
+            {this.props.incident.dangerous_occurrence_subparagraph_ids.length >= 1 ? (
+              <div className="block">
+                {this.props.incident.dangerous_occurrence_subparagraph_ids.map((code) => (
+                  <Tag>{this.props.complianceCodesHash[code]}</Tag>
+                ))}
+              </div>
+            ) : (
+              <p>{Strings.EMPTY_FIELD}</p>
+            )}
+          </div>
+        )}
+        <div className="inline-flex padding-small">
+          <p className="field-title">Who made the determination?</p>
+          <p>
+            {this.props.inspectorsHash[this.props.incident.determination_inspector_party_guid] ||
+              Strings.EMPTY_FIELD}
+          </p>
+        </div>
+      </div>
+      <br />
+    </div>
+  );
+
+  renderInitialDocuments = () => {
+    const initialDocuments = this.props.incident.documents.filter(
+      ({ mine_incident_document_type_code }) => mine_incident_document_type_code === "INI"
```

Comment:
Thoughts on pros/cons of setting this type code to a constant? It smells a bit of a magic string here, but if it's hyper-specific to just this component then it might not make sense to use a constant.